backend/app/verification.py

```diff
@@ -884,6 +912,78 @@ def unverify(
         worker.republish_app.send(app_id)
 
 
+@router.post("/{app_id}/enroll", status_code=204, tags=["verification"])
+def enroll(
+    login=Depends(logged_in),
```

Comment:
This seems basic, shouldn't we check if the user actually is the dev of this app?

You can use `Depends(app_author_only)` for that btw

Reply:
Done.